mitosheet/src/utils/actions.tsx

```diff
@@ -1281,11 +1302,32 @@ export const createActions = (
             searchTerms: ['CodeSnippets'],
             tooltip: "CodeSnippets"
         },
+        [ActionEnum.EXPORT_TO_FILE]: {
+            type: ActionEnum.EXPORT_TO_FILE,
```

Comment:
This action name and titles are pretty ambiguous. I think this should be about generating the code. Right now if you search for export, it's very confusing! 

<img width="400" alt="Screenshot 2023-01-17 at 5 53 00 PM" src="https://user-images.githubusercontent.com/18709905/213029778-501b2ae0-f52b-411f-8ad0-da8b664f4e03.png">